react-graphql-apollo-tutorial.md

```diff
@@ -2756,15 +2752,17 @@ const IssueItem = ({ issue }) => (
 export default IssueItem;
 {{< /highlight >}}
 
-Once you start your application again, you should see the initial page of paginated issues rendered below of each repository. That's the performance bottleneck which was mentioned earlier. It is even worse, because all GraphQL requests are execute on their own. They are not bundled in one request as they would have been when nesting the issues list field in the query next to the Organization and Profile components. However, in the next steps you are implementing a client-side filtering. The default is to show no issues at all. But it can toggle between the states of showing none, showing open issues and showing closed issues by using a button. Thus the issues will not be queried before toggling to one of the issue states.
+Once you start your application again, you should see the initial page of paginated issues rendered below each repository. That's the performance bottleneck mentioned earlier. Worse, the GraphQL requests  are not bundled in one request, as with the issues list field in the Organization and Profile components. In the next steps you are implementing client-side filtering. The default is to show no issues, but it can toggle between states of showing none, open issues, and closed issues using a button, so the issues will not be queried before toggling one of the issue states.
```

Comment:
"That's the performance bottleneck mentioned earlier." <- is this still be mentioned earlier?

Reply:
I don't think it was, but I assumed you would add it.  I changed it to "That's a performance bottleneck" just to be sure.